server/s3_put_bucket_replication_action.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
+
+#pragma once
```

Comment:
can be placed inside #ifdef check for __S3_SERVER_S3_PUT_BUCKET_REPLICATION_ACTION_H__.

Reply:
#pragma is used to ensure the header file gets included only once.
Referred other APIs files, they are having same format. i.e outside #ifdef

Reply:
Actually #ifdef check also does similar thing. No problem.